tests/Feature/Api/V1/UserSetRequestsTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use App\Models\UserGameListEntry;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class UserSetRequestsTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+
+    /**
+     * Test that the API returns an empty response for a non-existent user.
+     */
+    public function testGetUserSetRequestsUnknownUser(): void
+    {
+        $this->get($this->apiUrl('GetUserSetRequests', ['u' => 'nonExistant']))
+            ->assertSuccessful()
+            ->assertJson([]);
+    }
+
+    /**
+     * Test that the API returns the correct set requests for an existing user.
+     */
+    public function testGetUserSetRequests(): void
+    {
+        $system = System::factory()->create();
+        $game = Game::factory()->create([
+            'ConsoleID' => $system->ID,
+            'ImageIcon' => '/Images/001234.png',
+        ]);
+
+        $user = User::factory()->create([
+            'RAPoints' => 1501, // enough points to set request total to 1
+        ]);
+
+        // Create a user game list entry for the set request
+        UserGameListEntry::factory()->create([
+            'user_id' => $user->ID,
+            'type' => 'achievement_set_request',
+            'GameID' => $game->ID,
+        ]);
+
+        $userRequestInfo = [
+            'total' => 1,
+            'pointsForNext' => 999,
+        ];
```

Comment:
nit: Mental load for this test case could be reduced by deleting this variable and just hardcoding the values into the assert.

Reply:
Good catch, this was left over from when I was trying to figure out how to get predictable results from the User creator. Removed in latest